airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/CdcMssqlSourceDatatypeTest.java

```diff
@@ -391,32 +391,27 @@ protected void initTests() {
             .sourceType("datetimeoffset")
             .airbyteType(JsonSchemaType.STRING)
             .addInsertValues("'0001-01-10 00:00:00 +01:00'", "'9999-01-10 00:00:00 +01:00'", "null")
-            .addExpectedValues("0001-01-10 00:00:00 +01:00", "9999-01-10 00:00:00 +01:00", null)
+            .addExpectedValues("0001-01-10 00:00:00.0000000 +01:00",
```

Comment:
why do we expect to have milliseconds when insert values do not have them?

Reply:
I thought this was your concern in the comment above https://github.com/airbytehq/airbyte/pull/14449#pullrequestreview-1030820583
Previously we had millis in any case. I removed it for cases when source didn't have it. I thought you had a concern about it so I added it back, but aligned it to the format that we have for other connectors. And also aligned between normal and CDC cync. Because for now we have different behavior even inside mssql connector.  If it's not OK, kindly ask to specify your expected behavior. Thanks